galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager.ivt/src/main/java/dev/galasa/core/manager/ivt/CoreManagerIVT.java

```diff
@@ -27,6 +35,10 @@
     @CoreManager
     public ICoreManager coreManager;
     
+    @TestProperty(prefix="random", suffix="ivt", infixes="property.core")
+    public String prop;  
+    
+    
     @Test
     public void checkLogger() throws Exception {
         if (logger == null) {
```

Comment:
Can we change this to use a assertion please

Reply:
if you mean checkLogger then it is not I who wrote those methods and I did not wanted to touch them, but I can do that if you and @Michael-Baylis want me to.

Reply:
Yes please. When this test was written originally we didn't have they assert library in the framework. We should update the tests as we go

Reply:
Ok I will do it then. Is the logger still needed for those methods then?

Reply:
Not really. If the assertion fails the. We get a good message